### PR TITLE
Develop->Master - Theme version update / robots.txt liquid fix

### PR DESCRIPTION
**Pull request by kylekirkby**
Develop->Master - Theme version update / robots.txt liquid fix

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-{% if site.production == true %}
+{% if site.production %}
 User-agent: * 
-Disallow:
 Sitemap: {{site.url}}/sitemap.xml
 {% else %}
 User-agent: *
 Disallow: /
-Sitemap: {{site.url}}/sitemap.xml
 {% endif %}
```
Comment (morancj) [inline robots.txt]: `---
{% if site.production %}
User-agent: *
Sitemap: {{site.url}}/sitemap.xml
{% else %}
User-agent: *
Disallow: /
{% endif %}

`

File: robots.txt [hunk 2]
```diff
@@ -1,11 +1,9 @@
 ---
```
Comment (morancj) [inline robots.txt]: ``

Comment (morancj) [inline robots.txt]:
```suggestion
```

File: robots.txt [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
 ---
-{% if site.production == true %}
+{% if site.production %}
 User-agent: *
```
Comment (morancj) [inline robots.txt]:
```suggestion
User-agent: *
```

Comment (kylekirkby) [reply robots.txt]: What are you suggesting here?

**Comment by morancj** [comment]
Sorry, wrong form of comment. Give me a couple minutes... :smiley:

File: robots.txt (hunk 2 above)
Comment (morancj) [reply robots.txt]: Two header lines?

File: robots.txt (hunk 3 above)
Comment (morancj) [reply robots.txt]: Trailing whitespace :)

**Comment by morancj** [comment]
Hi @kylekirkby ,
What's processing the `robots.txt` file? It looks like [Jinja(2)](https://jinja.palletsprojects.com/en/2.10.x/templates/#if).
If I leave the `---` line in, the `if` statement works, but the `---` line remains in the processed output: if I remove the line, no processing occurs.

**Comment by kylekirkby** [comment]
> Hi @kylekirkby ,
> What's processing the `robots.txt` file? It looks like [Jinja(2)](https://jinja.palletsprojects.com/en/2.10.x/templates/#if).
> If I leave the `---` line in, the `if` statement works, but the `---` line remains in the processed output: if I remove the line, no processing occurs.

Jekyll uses the liquid templating language by Shopify. See details here:
https://jekyllrb.com/docs/liquid/
The
```
---
---
```
at the top of the file tells Jekyll to process the liquid in the file (not sure where I found that trick as I can't see it documented)

**Comment by morancj** [comment]
Thanks @kylekirkby , learn something new every day! I tried the double header line, and it works as you describe. :smile:

**Pull request merged**